src/Sentry.Unity.Editor/Android/GradleSetup.cs

```diff
@@ -1,21 +1,18 @@
-using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Linq;
+using System.Text.RegularExpressions;
 using Sentry.Extensibility;
-using UnityEditor.Build;
 
 namespace Sentry.Unity.Editor.Android
 {
     internal class GradleSetup
     {
         private readonly IDiagnosticLogger _logger;
 
-        public const string DependencyScopeName = "dependencies";
-        public const string SdkDependencies = @"implementation(name: 'sentry-android-ndk-release', ext:'aar')
-    implementation(name: 'sentry-android-core-release', ext:'aar')";
-        public static readonly List<string> ScopesToSkip = new() { "buildscript", "pluginManagement" };
+        private const string AndroidMarker = "android {";
+        private const string SdkDependenciesFull = SdkDependencies + "\n}\n\n" + AndroidMarker;
 
+        public const string SdkDependencies = "dependencies {\n\timplementation(name: 'sentry-android-ndk-release', ext:'aar')\n\timplementation(name: 'sentry-android-core-release', ext:'aar')";
```

Comment:
```suggestion
        public const string SdkDependencies = @"dependencies {
            implementation(name: 'sentry-android-ndk-release', ext:'aar')
            implementation(name: 'sentry-android-core-release', ext:'aar')";
```
Just do stay consistent with other places.
The block does not need a closing `}`?

Reply:
I see, because we're just replacing the android marker. Makes sense.
I'm still confused about replacing `android {` with `dependencies {`.